pkg/config/lima_config_applier.go

```diff
@@ -17,6 +17,8 @@ import (
 )
 
 const userModeEmulationProvisioningScriptHeader = "# cross-arch tools"
+const sociInstallationProvisioningScriptHeader = "# soci installation and configuring"
+const sociVersion = "0.1.0"
```

Comment:
```suggestion
const sociVersion = "0.3.0"
```

https://github.com/awslabs/soci-snapshotter/releases/tag/v0.3.0

Reply:
I'm not sure why but when I change ```sociVersion``` to anything above ```0.1.0``` locally it causes issues with the installation. I'll keep looking into this though and figure it out.

Reply:
`0.3.0` has to be used here. `0.1.0` doesn't work with your nerdctl code.